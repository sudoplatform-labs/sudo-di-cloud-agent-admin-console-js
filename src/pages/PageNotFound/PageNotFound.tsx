import React from 'react';
import { SimplePage } from '../../components/SimplePage';

export const PageNotFound = (): JSX.Element => (
  <SimplePage>
    <h1>Page not found</h1>
  </SimplePage>
);
