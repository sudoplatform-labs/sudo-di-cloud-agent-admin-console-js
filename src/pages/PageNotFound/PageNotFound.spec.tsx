import { shallow } from 'enzyme';
import React from 'react';
import { PageNotFound } from './PageNotFound';

describe('pages/EditProfile/EditProfile', () => {
  it('should render', () => {
    const component = shallow(<PageNotFound />);
    expect(component).toMatchSnapshot();
  });
});
