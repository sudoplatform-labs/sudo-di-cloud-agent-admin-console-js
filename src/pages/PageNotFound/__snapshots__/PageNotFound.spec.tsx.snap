// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/EditProfile/EditProfile should render 1`] = `
<SimplePage>
  <h1>
    Page not found
  </h1>
</SimplePage>
`;
